ban/commands/ignsna.py

```diff
@@ -0,0 +1,48 @@
+import os
+from ban.commands import command, report
+from ban.core.models import (Municipality, PostCode)
+from .helpers import session, batch, nodiff
+
+__namespace__ = 'import'
+
+
+@command
+@nodiff
+def ignsna(path, **kwargs):
+    """Import from IGN/Laposte BDUNI
+    :param path: directory location of the files hsp7aaaa.ai, hsv7aaaa.ai and hsw4aaaa.ai"""
+
+    municipality_zipcode_file = os.path.join(path, 'hsp7aaaa.ai')
+    if municipality_zipcode_file is not None:
+        lines = open(municipality_zipcode_file).readlines()
```

Comment:
But to have the length of f, I must compute this or I use "cardinality" dependency like this :
```python
    cardinality.count(f)
``` 

And
```python
    with open(zipcode_file) as f:
        m_dir = cardinality.count(f)
        batch(process_postcode_file, f, max_value= m_dir)
```
return done without the traitment.
Maybe I make a mistake